packages/connect-explorer-nextra/package.json

```diff
@@ -26,6 +26,7 @@
         "@trezor/theme": "workspace:^",
         "@trezor/utils": "workspace:^",
         "babel-plugin-styled-components": "^2.1.4",
+        "concurrently": "^8.2.2",
```

Comment:
nit: shouldn't this go to dev deps? also babel, and other similar libs?

Reply:
Yes you're right, I'll look into it